### PR TITLE
Deprecate '@adeira/test-utils'

### PR DESCRIPTION
**Pull request by mrtnzlml**
I was thinking for a while what to do with `@adeira/test-utils` package. There are only two functions (`evaluateGraphQLResolver` and `generateTestsFromFixtures`) the GraphQL resolver evaluation seems to be pretty useless/outdated. On the other hand the fixtures test generator is quite popular and helpful.

I'd like to propose a deprecation of https://www.npmjs.com/package/@adeira/test-utils and I'd like to create a new NPM package instead which is specializing only on the fixtures testing.

### CODE REVIEW

File: src/graphql-resolve-wrapper/package.json [hunk 1]
```diff
@@ -4,15 +4,12 @@
...
   "private": false,
-  "version": "0.2.0",
+  "version": "0.3.0",
   "main": "src/index",
   "sideEffects": false,
   "dependencies": {
     "@babel/runtime": "^7.12.5"
   },
-  "devDependencies": {
-    "@adeira/test-utils": "^0.2.0"
```
Comment (mrtnzlml) [inline src/graphql-resolve-wrapper/package.json:14]: Even we were using some prehistoric version of the GraphQL resolver tester.

File: yarn.lock [hunk 2]
```diff
@@ -2,20 +2,6 @@
...
-  dependencies:
-    "@babel/runtime" "^7.11.2"
-
-"@adeira/test-utils@^0.2.0":
-  version "0.2.0"
-  resolved "https://registry.yarnpkg.com/@adeira/test-utils/-/test-utils-0.2.0.tgz#9e9feef7bb10968d161dcd4a568e760af25b7c50"
-  integrity sha512-RhnDtUQMNmhzQyGtjewXckGtBvn15qNOnNrIy5fnLTAY1vjowjglKpB+AXl2eq7xpkf+2k6R4J7v06NYU2HmQw==
-  dependencies:
-    "@adeira/js" "^1.2.0"
-
```
Comment (mrtnzlml) [inline yarn.lock:18]: No `@adeira` packages should ever be in `yarn.lock`.

Comment (tbergq) [reply yarn.lock:18]: Could we add some scanner test for this maybe?

**Review by tbergq — APPROVED** [review]

File: yarn.lock (hunk 2 above)
Comment (mrtnzlml) [reply yarn.lock:18]: Added into wishlist: https://github.com/adeira/universe/issues/1301

**Pull request merged**